storage/innobase/dict/dict0dict.cc

```diff
@@ -906,38 +906,36 @@ dict_index_get_nth_col_or_prefix_pos(
 }
 
 /** Returns TRUE if the index contains a column or a prefix of that column.
-@param[in]	index		index
 @param[in]	n		column number
 @param[in]	is_virtual	whether it is a virtual col
 @return TRUE if contains the column or its prefix */
 bool
-dict_index_contains_col_or_prefix(
-	const dict_index_t*	index,
+dict_index_t::contains_col_or_prefix(
 	ulint			n,
-	bool			is_virtual)
+	bool			is_virtual) const
```

Comment:
It depends on `dict_table_get_nth_col()`/`dict_table_get_nth_v_col()`.

Reply:
There is many calls that use `(i, is_virtual)` interface. I don't think it is good idea to change to `dict_col_t` (especially by reference), because callers don't have it.